tests/data/nyaml2nxdl/NXellipsometry-docCheck.yaml

```diff
@@ -1,588 +1,543 @@
-#01/2022
-#a draft version of a NeXus application definition for ellipsometry
-
-# the document has the following main elements:
-# instrument used and is characteristics
-# measured data, the discription of the sample and what was measured about it
-# derived parameters: extra parameters derived in the measurement software
-
-# one would need a Class/ separate entry for the sample description which can link to and should contain a full description of the composition.
-# A free description as text might be not enough or unclear. Especially if one deals with many layer structured samples and optically anisotropic
-# samples, like triclinic one
-#
-# Comments are moved to the Development_notes.md file
```

Comment:
do you plan to put comments back when the comment preservation is implemented?

Reply:
Yes, I have plan for that.